docs/make.bat

```diff
@@ -46,15 +45,14 @@ if "%1" == "clean" (
 	for /d %%i in (%BUILDDIR%\*) do rmdir /q /s %%i
 	del /q /s %BUILDDIR%\*
 	for /d %%i in (source\*) do rmdir /q /s %%i
-
+	del /q /s source\extensions.rst
```

Comment:
could we maybe use the full package name of `jupyter_contrib_nbextensions` here too? Or at least specify `nbextensions` over `extensions`